internal/services/dynatrace/helper.go

```diff
@@ -0,0 +1,398 @@
+package dynatrace
+
+import (
+	"github.com/hashicorp/go-azure-sdk/resource-manager/dynatrace/2021-09-01/monitors"
+	"github.com/hashicorp/go-azure-sdk/resource-manager/dynatrace/2021-09-01/tagrules"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/tf/pluginsdk"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/tf/validation"
+)
+
+func SchemaPlanData() *pluginsdk.Schema {
```

Comment:
why are these exrtacted into functions and not inline?

Reply:
This is for future reuse.